express-static/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const test = path.join(__dirname, 'public');
```

Comment:
Not a huge deal for the exercise but make sure to use descriptive variable names. Here something like `staticMiddleware` would be good